tools/MemoryCheckstyle.xml

```diff
@@ -395,4 +396,4 @@ under the License.
 
   </module> <!-- End of TreeWalker -->
 
-</module>
+</module>
```

Comment:
This file has been updated to reflect the same configuration as present in datasketches-java.